dts/bindings/clock/nuvoton,numaker-scc.yaml

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2023 Nuvoton Technology Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: Nuvoton NuMaker System Clock Controller (SCC)
+
+compatible: "nuvoton,numaker-scc"
+
+include: [clock-controller.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  hxt:
```

Comment:
why not make this a boolean,  eg `hxt-enable`? Same for others.

Reply:
Thanks of your advice. Reserve for one more choice "untouched".